test/testutil.py

```diff
@@ -108,3 +109,37 @@ def deregister_ami(ami_id):
         err_msg = err.response["Error"]["Message"]
         print(f"Couldn't deregister image {ami_id}.")
         print(f"Error {err_code}: {err_msg}")
+
+
+def make_fake_tree(basedir: pathlib.Path, fake_content: dict):
```

Comment:
Those two look like they come from osbuild which is totally fine, but it would be nice to add a comment where they come from. Importing from osbuild would be even nicer but that is probably a bit overkill (or a shared testutil repo which is equally overkill)

Reply:
Yeah I thought about that too. But it did seem like overkill. I will add comments, like you suggested, pointing back to the original code